src/user-icon/classes.scss

```diff
@@ -6,6 +6,10 @@
   @include iui-user-icon;
   @include iui-user-icon-size;
 
+  &#{&} {
```

Comment:
Is it the same as writing the same class? 😮

Reply:
It is 😄 

`&` refers to the selector (`.iui-user-icon` class) and `#` is needed to concatenate